.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+localConfig.yml
```

Comment:
```suggestion
docker/localConfig.yml
```

Correct?

Reply:
this is more accurate yes, both are valid. Not specifying the directory will just ignore all occurrences of this file name